docs/en/ingest-management/fleet/add-fleet-server-kubernetes.asciidoc

```diff
@@ -0,0 +1,392 @@
+[[add-fleet-server-kubernetes]]
+= Deploy Fleet Server on Kubernetes
+
+[NOTE]
+====
+For ECK users, if your {stack} is orchestrated by ECK, we recommend to orchestrate the {fleet-server} through the operator. That simplifies the process as all needed resources will be configured by ECK.
++
```

Comment:
It's a bit confusing, but you don't need a `+` here. This actually renders in the output in this instance.